images/fluent-bit/tests/02-helm-install.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+
+# monopod:tag:functional
+
+set -o errexit -o nounset -o errtrace -o pipefail -x
+
+IMAGE_DIR="$(basename "$(cd "$(dirname ${BASH_SOURCE[0]})/.." && pwd )")"
+IMAGE_NAME=${IMAGE_NAME:-"cgr.dev/chainguard/${IMAGE_DIR}:latest"}
```

Comment:
Do we want to make the default for functional tests registry.local, that way we don't end up inadvertently testing an already published image if IMAGE_NAME is unavailable for some reason?

Reply:
done!